modules/indexer/issues/meilisearch/meilisearch.go

```diff
@@ -247,73 +255,31 @@ func parseSortBy(sortBy internal.SortBy) string {
 	return field + ":asc"
 }
 
-// nonFuzzyWorkaround is needed as meilisearch does not have an exact search
-// and you can only change "typo tolerance" per index. So we have to post-filter the results
-// https://www.meilisearch.com/docs/learn/configuration/typo_tolerance#configuring-typo-tolerance
-// TODO: remove once https://github.com/orgs/meilisearch/discussions/377 is addressed
-func nonFuzzyWorkaround(searchRes *meilisearch.SearchResponse, keyword string, isFuzzy bool) ([]internal.Match, error) {
+func doubleQuoteKeyword(k string) string {
+	kp := strings.Split(k, " ")
+	parts := 0
+	for i := range kp {
+		if kp[i] != "" {
+			kp[parts] = fmt.Sprintf(`"%s"`, kp[i])
```

Comment:
Maybe we should check whether it already has double quotes.

Reply:
The last one from me, otherwise LGTM.

Reply:
done -> 6b9f4ebe8bc8a21805fc20513b63a85225801ed5